src/public/locales/ig/common.json

```diff
@@ -1,5 +1,5 @@
 {
-  "The First African Language ": " mbụ nwere asụsụ si Afrịka",
+  "The First African Language API": "$API$ mbụ nwere asụsụ si Afrịka",
```

Comment:
we exclude API in this string to be able to apply styles. you can remove API here!

Reply:
In the geeman tranalation the word "API" is in the middle of the sentence, so we can't append or prepend the styled word like before,so i added "$API$"  to signal where it would be styled. Now it is practically included in the translation string, so wluldn't it make sense to keep it in the key?

Reply:
How would that affect the English and Igbo translations? Would the API keyword be misplaced with those translations?

Reply:
If you look in src/pages/App.js on my fork, you will see, how i handle that. this,of course isn'tideal, as it only, specifically handles this one case. But i think its OK for now.